ReactQt/runtime/src/reactview.cpp

```diff
@@ -109,8 +110,19 @@ private Q_SLOTS:
         if (bridge != nullptr && bridge->ready())
             monitorChangeUrl();
     }
+
+    void codeLocationChanged();
 };
 
+void ReactViewPrivate::codeLocationChanged() {
```

Comment:
I would suggest to use more meaningful name on slots that describe functionality, for example `resetBundleUrl()`.
For cases, when signal should result in non-related actions, slot can be called `onCodeLocationChanged()` to distinguish it from signal during textual search.

Also, in this particular case all statements work with `Bridge` object, so we can make it a slot of `Bridge` class - `Bridge::setBundle()`. Maybe we will need to invoke it from another places in the code.